code/game/objects/items/hot_potato.dm

```diff
@@ -172,3 +173,4 @@
 	sticky = FALSE
 	reusable = TRUE
 	forceful_attachment = FALSE
+	stimulant = FALSE
```

Comment:
wait, hold on, this is different from what you said in the PR's body.

instead of removing muscle stimulant from the allergy chem pool (by, say, reclassifying it as a drug instead of a medicine), you've just papered over the problem by making toy hot potatoes not dose you with stimulants. traitor mime hot potatoes, for example, will still be able to trigger muscle stimulant allergies. in addition, rolling hot potato muscle stimulants as one of your allergies is basically a "dud"/"free space" roll now.

Reply:
I thought about it for a little bit before submitting the pull, and the change to the toy potatoes to not stim was what I felt was _most_ appropriate, leaving that rare edge case of a traitor hot potato still deleting someone. Not that I think it's ever been abused currently, but I feel like having an arcade prize that allows someone to dose themselves with effectively infinite muscle stims to ignore damage slows is not the best idea.

Regarding the allergy free roll, there are several of reagents that it can roll that are of zero consequences to the player. The only blacklisted medicines are epinephrine, adminordrazine, godblood, coriolis hepatico, diphenhydramine, and synaphydramine. Other noteworthy allergy rolls that I've seen are things like quantum medicine, changeling adrenaline, experimental stimulants -- these are almost always free rolls as well.

Reply:
This looks fine to me.